views/default/input/date.php

```diff
@@ -48,8 +52,20 @@
 	$vars['value'] = gmdate('Y-m-d', $vars['value']);
 }
 
+$datepicker_options = elgg_extract('datepicker_options', $vars);
```

Comment:
I am afraid if someone passes `false` we would end up with problems in JS `$.extend`